src/exampleTypes/TagsQuery/ActionsMenu.js

```diff
@@ -7,7 +7,13 @@ import { withTheme } from '../../utils/theme'
 import { Flex } from '../../greyVest'
 import { copyTags, tagTerm } from './utils'
 
-let ActionsMenu = ({ node, tree, open, theme: { Button, Checkbox } }) => (
+let ActionsMenu = ({
+  node,
+  tree,
+  open,
+  theme: { Button, Checkbox },
+  actionWrapper = _.noop,
```

Comment:
I think should default to `_.identity`. If you omit `actionWrapper`, `_.noop` returns undefined which would prevent the onClicks from doing anything.